index.js

```diff
@@ -0,0 +1,248 @@
+const ProjectsMobile = [{
+  Title1: 'My recent works',
+  Image: 'assets/images/Placeholder.png',
+  Title2: 'Multi-Post Stories',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the      industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+  Tecnologies: ['CSS', 'HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+},
+{
+  Name: 'Profesional Art Printing Data',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+},
+{
+  Title1: 'Multi-Post Stories',
+  Close: '&times;',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Image: ['/assets/Portfolio.png', 'popup image'],
+  Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  Links: [['https://www.google.com/', 'see live', '/assets/Icon.png', 'google'], ['https://github.com/', 'see source', '/assets/Vector.png', 'github']],
+},
+{
+  Linkk: 'See Project',
+},
+{
+  Titled1: 'Data Dashboard Healthcare',
+  Titlew1: 'Website Protfolio',
+  Titlepro: 'Profesional Art Printing Data More',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has  been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'See Project',
+}];
```

Comment:
- [ ] Nice work making the changes :+1: However it would be nice to store all the projects with the same properties. That will help you to easily loop through the array and display the projects dynamically.

- [ ] Kindly make sure that each object contains at least the following information about the project: name or title, description, image, technologies, link to live version, and link to the project. As requested in the project requirement.
**E.g. Your array could look like this :point_down:**
```javascript
const ProjectsMobile = [
{
    name: "Title of project 1",
    description: "Description of project 1",
    image: "Link to project 1 images",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: ["Link_to_live_version", "Link_to_repo"]
},
{
    name: "Title of project 2",
    description: "Description of project 2",
    image: "Link to project 2 images",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: ["Link_to_live_version", "Link_to_repo"]
},
{
    name: "Title of project 3",
    description: "Description of project 3",
    image: "Link to project 3 images",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: ["Link_to_live_version", "Link_to_repo"]
},
...
]
```

_These suggestions are required because you make changes to this array after the first review_